klock/src/commonTest/kotlin/korlibs/time/FrequencyTest.kt

```diff
@@ -9,4 +9,16 @@ class FrequencyTest {
         assertEquals(100.milliseconds, 10.timesPerSecond.timeSpan)
         assertEquals(10.hz, 100.milliseconds.hz)
     }
+
+    @Test
+    fun `test frequency operators work as expected`() {
```

Comment:
This might not work, since on JS and Android IIRC this might not be valid. Let's wait for CI

Reply:
Done.